packages/expo-cli/src/commands/codesigning/configureCodeSigningAsync.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  convertCertificatePEMToCertificate,
+  convertKeyPairPEMToKeyPair,
+  validateSelfSignedCertificate,
+} from '@expo/code-signing-certificates';
+import { ExpoConfig, getConfig } from '@expo/config';
+import assert from 'assert';
+import { promises as fs } from 'fs';
+import path from 'path';
+
+import Log from '../../log';
+import { attemptModification } from '../utils/modifyConfigAsync';
+
+type Options = { input?: string };
+
+export async function actionAsync(projectRoot: string, { input }: Options) {
+  assert(typeof input === 'string', '--input must be a string');
+
+  const inputDir = path.resolve(projectRoot, input);
+
+  const [certificatePEM, privateKeyPEM, publicKeyPEM] = (
+    await Promise.all(
+      ['certificate.pem', 'private-key.pem', 'public-key.pem'].map(fname =>
+        fs.readFile(`${inputDir}/${fname}`)
+      )
+    )
+  ).map(buffer => buffer.toString());
+
+  const certificate = convertCertificatePEMToCertificate(certificatePEM);
+  const keyPair = convertKeyPairPEMToKeyPair({ privateKeyPEM, publicKeyPEM });
+  validateSelfSignedCertificate(certificate, keyPair);
+
+  const { exp } = getConfig(projectRoot, { skipSDKVersionRequirement: true });
+
+  const fields: ExpoConfig['updates'] = {
+    codeSigningCertificate: `./${input}/certificate.pem`,
+    codeSigningMetadata: {
+      keyid: 'main',
+      alg: 'rsa-v1_5-sha256',
+    },
+  };
+  await attemptModification(
+    projectRoot,
+    {
+      updates: {
+        ...(exp.updates || {}),
```

Comment:
```suggestion
        ...exp.updates,
```
It's fine if it's null/undefined